edit/edit.go

```diff
@@ -552,8 +578,13 @@ func match(ed *Editor, at addr, re *re1.Regexp) ([][2]int64, error) {
 // 		stands for the string that matched the d-th subexpression.
 // 		It is an error if such a subexpression has more than MaxRunes.
 //		\n is a literal newline.
+//		A number after s indicates we substitute the Nth match in the
+//		address range.
 // 		If the delimiter after the text is followed by the letter g
 //		then all matches in the address range are substituted.
+//		If a number (n) and the letter g are both present then the Nth
```

Comment:
Can you change the command syntax above to account for n?
It should be `{addr} s{n}/regexp/text/`